assignment 6/day 7/index.js

```diff
@@ -0,0 +1,29 @@
+const fs = require('fs');
+
+function readFile(file) {
+  return new Promise(function(resolve, reject) {
+    fs.readFile(file, 'utf8', function(err, data) {
+      if (err) return reject(err);
```

Comment:
the statement in if block should come in new-line and if block must always have emclosing `{}`, even if it is a single statement